src/bindings/python/src/openvino/frontend/pytorch/module_extension.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# flake8: noqa
+# mypy: ignore-errors
+
+class ModuleExtension:
```

Comment:
```suggestion
class PyTorchModuleExtension:
```

Reply:
In TensorFlow we also have tf.Module: https://www.tensorflow.org/api_docs/python/tf/Module